### PR TITLE
[15.0][IMP] database_cleanup: purge fields (forward-port)

### PR DESCRIPTION
**Pull request by astirpe**
This is the forward-port of https://github.com/OCA/server-tools/pull/2491
with the addition of some tests.

### CODE REVIEW

File: database_cleanup/tests/test_purge_fields.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import tagged
+
+from .common import Common, environment
+
+
+# Use post_install to get all models loaded more info: odoo/odoo#13458
+@tagged("post_install", "-at_install")
+class TestCleanupPurgeLineColumn(Common):
```
Comment (ypapouin) [inline database_cleanup/tests/test_purge_fields.py]: Unrelated classname should be `TestCleanupPurgeFields`

**Review by CasVissers-360ERP — APPROVED** [review]
Functional review

File: database_cleanup/tests/test_purge_fields.py (hunk 1 above)
Comment (astirpe) [reply database_cleanup/tests/test_purge_fields.py]: Right, thanks!

**Review by thomaspaulb — APPROVED** [review]

**Comment by CasVissers-360ERP** [comment]
@astirpe can you forward port to v16/v17?

**Pull request merged**